upgrade.yml

```diff
@@ -92,7 +92,7 @@
       service: name=mysql state=restarted
 
 - name: upgrade rabbitmq
-  gather_facts: force
+  # gather_facts: force
```

Comment:
This file is going to change out from under you, so best to just leave it alone. I'll deal with 2.0 changes when this PR lands (or if it lands before the upgrade PR, I'll do it in the upgrade PR).